### PR TITLE
fixing organization download checkboxes

### PR DESCRIPTION
**Pull request by calellowitz**
@benrudolph @kaapstorm cc: @proteusvacuum 
fixes http://manage.dimagi.com/default.asp?190517#1068279

### CODE REVIEW

File: corehq/apps/commtrack/static/commtrack/ko/location_bulk_upload_file.js [hunk 1]
```diff
@@ -15,9 +15,10 @@ $(function () {
         self = this;
         this.url = ko.computed(function() {
             return (
-                self.base_url
-                + (self.include_consumption() ? "?include_consumption=true" : "")
-                + (self.include_ids() ? "?include_ids=true" : "")
+                self.base_url + "?"
+                + (self.include_consumption() ? "include_consumption=true" : "")
```
Comment (dimagimon) [inline corehq/apps/commtrack/static/commtrack/ko/location_bulk_upload_file.js]: Bad line breaking before '+'.

File: corehq/apps/commtrack/static/commtrack/ko/location_bulk_upload_file.js [hunk 2]
```diff
@@ -15,9 +15,10 @@ $(function () {
...
+                + ((self.include_consumption() && self.include_ids()) ? "&" : "")
```
Comment (dimagimon) [inline corehq/apps/commtrack/static/commtrack/ko/location_bulk_upload_file.js]: Bad line breaking before '+'.

File: corehq/apps/commtrack/static/commtrack/ko/location_bulk_upload_file.js [hunk 3]
```diff
@@ -15,9 +15,10 @@ $(function () {
...
+                + (self.include_ids() ? "include_ids=true" : "")
```
Comment (dimagimon) [inline corehq/apps/commtrack/static/commtrack/ko/location_bulk_upload_file.js]: Bad line breaking before '+'.

**Comment by benrudolph** [comment]
no test coverage

**Pull request merged**